american-journal-of-archaeology.csl

```diff
@@ -320,14 +337,13 @@
         <text macro="day-month" prefix=", "/>
       </else-if>
       <else>
-        <group prefix=". " delimiter=", ">
+        <group prefix="" delimiter=", ">
```

Comment:
I don't understand the problem with the period in the prefix here? I'm seeing a period before Ph.D. diss in the style guide and the solution with `text value` above is really awkward.

Reply:
I believe this is because it is preceded by the title of the dissertation which already ends with a period, so if you include it here too, you get things like `"Title of my dissertation." . Ph.D., Alma Mater University.` .

If there's a better way to handle it, great.